api/src/test/java/org/openmrs/test/StartModuleExecutionListener.java

```diff
@@ -9,6 +9,8 @@
  */
 package org.openmrs.test;
 
```

Comment:
Since we have to support JUnit 4 tests for a while and this is used in the `BaseContextSensitiveTest` helper class we have to leave this one as it is and create a JUnit 5 version (which I am currently doing). Please remove this file from the PR and I will merge it :)

Reply:
thanlks